fdb-record-layer-spatial/src/main/java/com/apple/foundationdb/record/spatial/geophile/GeophileBoxLatLon.java

```diff
@@ -57,16 +57,19 @@ public static SpatialObject newBox(double latLo,
                 ? new Box(latLo, latHi, lonLo, lonHi)
                 : new GeophileBoxLatLonWithWraparound(latLo, latHi, lonLo, lonHi);
         } catch (IllegalArgumentException ex) {
-            throw new RecordCoreArgumentException(String.format("cannot make box: latLo = %s, latHi = %s, lonLo = %s, lonHi = %s",
-                                                                latLo, latHi, lonLo, lonHi), ex);
+            throw new RecordCoreArgumentException("cannot make box", ex)
+                    .addLogInfo("latLo", latLo)
+                    .addLogInfo("latHi", latHi)
+                    .addLogInfo("lonLo", lonLo)
+                    .addLogInfo("lonHi", lonHi);
         }
     }
 
     // Query boxes are specified as center point +- delta, where delta <= 360.
     // This calculation can put us past min/max lon.
     private static double fixLon(double lon) {
         if (lon > GeophileSpatial.MAX_LON + CIRCLE || lon < GeophileSpatial.MIN_LON - CIRCLE) {
-            throw new RecordCoreArgumentException(String.format("longitude %s", lon));
+            throw new RecordCoreArgumentException("longitude").addLogInfo("longitude", lon);
```

Comment:
This exception and the one below do not have great messages, but I'm not remotely familiar with this code, so I don't know what they should be... Perhaps simply `Invalid value for longitude` since it is just comparing to constants.

Reply:
I've added the word "invalid", though maybe it's more like "out of range"? I'm not sure how much to workshop this